src/express.re

```diff
@@ -1,23 +1,43 @@
-let module Next = {
+/*
+  There are a few types to update in this file.
+  We're generic in 'a in most types, but really
+  should specify which JS type should be used.
+
+  An example of how to handle JS fns that take
+  more than one type, where some types are
+  containers and some primitives is the the
+  Response.send binding below.
+
+  sendString | sendObject | sendBuffer | sendArray
+  are all bindings to the .send method based on the
+  input type
+
+  tracked in https://github.com/BuckleTypes/Rexpress/issues/6
+ */
+module Next = {
   type t;
 };
 
-let module Request = {
+module Request = {
   type t;
 };
 
-let module Response = {
+module Response = {
   type t;
   external sendFile : t => string => 'a => unit = "" [@@bs.send];
-  external json: t => 'a => unit = "" [@@bs.send];
+  external sendString : t => string => unit = "send" [@@bs.send];
+  external sendObject : t => Js.Json.t => unit = "send" [@@bs.send];
```

Comment:
that's a good point. I had them separate in order to track both the `res.send` and `res.json` methods, but since they're both basically the same, I guess there's no reason not to just support `external json`. Shall I remove `sendObject` and then replace `json`'s signature with this signature in my separate PR?